### PR TITLE
Replace BCP14 boilerplate

### PR DESCRIPTION
**Pull request by becarpenter**
Just interpret the magic words as plain English, because this is not an interoperability spec. Did not apply bold face.

### CODE REVIEW

**Comment by larseggert** [comment]
Thanks for the PR. Do others think we should do this? (I'm on the fence.)

**Comment by russhousley** [comment]
I guess there is no harm.

**Comment by becarpenter** [comment]
The reason I'm advocating this change is that in the end we need wiggle room in IETF processes, and we can't accept wiggle room in protocol specs - so BCP14 is really the wrong instrument in process documents. (_All_ process documents, not just this one.)

File: draft-ecahc-moderation.md [hunk 1]
```diff
@@ -96,7 +96,10 @@ after, and subsumes, the moderator team for the IETF discussion list
 
 # Conventions and Definitions
 
-{::boilerplate bcp14-tagged}
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted in their normal English sense; they are shown
+in uppercase for emphasis and clarity.
```
[the remark this answers is not in the document]
Comment (becarpenter) [reply draft-ecahc-moderation.md:102]: Yes, good idea. (However, let's avoid bikeshedding this in case it becomes a distraction.)

**Comment by becarpenter** [comment]
PR now also includes note on preemptive vs reactive moderation in response to Joel's comment.

**Comment by larseggert** [comment]
One issue: if we leave these terms capitalized in the test, idnits will throw a hissy-fit. It might be better to lowercase and bold them?

**Comment by larseggert** [comment]
> PR now also includes note on preemptive vs reactive moderation in response to Joel's comment.

Could we do this in a separate PR, so we don't blur which PRs fix which issues?

**Comment by becarpenter** [comment]
1. re lowercase/bold. Good catch... I have no objection to that change.
2. Hmm. GitHub didn't seem to give me a way to make a separate PR. (I use GitHub Desktop on Windows.) So I agree, but I don't know how to do it.

**Comment by becarpenter** [comment]
I've taken the second item out of the PR, to avoid the confusion.

**Review by larseggert — APPROVED** [review]

**Review by larseggert — APPROVED** [review]

**Pull request merged**